torch/utils/data/datapipes/iter/tobytes.py

```diff
@@ -0,0 +1,21 @@
+from typing import Tuple
+from torch.utils.data import IterDataPipe
+
+
+class ToBytesIterDataPipe(IterDataPipe[Tuple[str, bytes]]):
+    r""" :class:`ToBytesIterDataPipe`
+
+    Iterable DataPipe to load IO stream with label name,
+    yield bytes with label name in a tuple
+    args:
+        max_limit : maximum bytes to read from stream
+    """
+    def __init__(self, source_datapipe, max_limit=None):
+        self.source_datapipe = source_datapipe
+        self.max_limit = max_limit
+
+    def __iter__(self):
+        for (furl, stream) in self.source_datapipe:
+            d = stream.read(self.max_limit)
+            stream.close()
```

Comment:
Please do not close stream, leave it to python's ref counters.

Reply:
got it. stream.close() is removed, relying totally on python's ref counters.
I test locally TestIterableDataPipeHttp.test_stress_http_reader_iterable_datapipes with 16M files and no issue is found. Given the default Linux IO handles max is 2M, if there is any unreleased OS handles, it would have raised the issue in the test.